### PR TITLE
Update FogHandler.java

### PR DESCRIPTION
**Pull request by TataTawa**
5% fps increase. It is not useful to calculate 1600 times per frame only to get newest fogx and fogz. 10% chance of this calculate per frame in another thread will be fine.

### CODE REVIEW

**Comment by Dream-Master** [comment]
@TataTawa are you on my Discord? What is you Discord Name?

**Comment by TataTawa** [comment]
oh i not use discord

File: src/main/java/biomesoplenty/client/fog/FogHandler.java [hunk 1]
```diff
@@ -22,6 +26,11 @@
 import org.lwjgl.opengl.GL11;
 
 import cpw.mods.fml.common.eventhandler.SubscribeEvent;
+import org.objectweb.asm.Opcodes;
```
Comment (draknyte1) [inline src/main/java/biomesoplenty/client/fog/FogHandler.java:29]: I don’t think you need this. Lul

**Comment by Dream-Master** [comment]
ok but beside this is the code ok ? @draknyte1

**Comment by draknyte1** [comment]
Yeah, but like anything that introduces threading, you want it to be tested (by more than one user using it long term)

Please make a note of any of the recent PRs which add such and be sure you get people to test these things next dev update.

**Comment by Dream-Master** [comment]
we will test it on zeta first

**Comment by Dream-Master** [comment]
@TataTawa can you remove unused import suggested by draknyte?

**Comment by Glease** [comment]
on hyper rare occasion, if ever, it would crash the client (but not the server) due to concurrent modification to biome cache. I'd say let's merge this for now and see this in action on zeta. We can put in remedy later if it turns out to be actually causing crash. The chance is fairly rare that I'd just pretend it is not doing that way.

Note for future: If any player submit a bug report with title like "Getting biome" and a ConcurrentModificationException then we move all these calculation back to main thread and reduce frequency further.

**Pull request merged**